src/xcode/ENA/ENA/Source/Scenes/HealthCertificate/Cells/HealthCertificateQRCodeCellViewModel.swift

```diff
@@ -42,5 +44,5 @@ struct HealthCertificateQRCodeCellViewModel {
 	let certificate: String
 	let validity: String
 	let qrCodeImage: UIImage
-
+	let accessibility: String?
```

Comment:
```suggestion
	let accessibilityLabel: String?
```

Reply:
It's not a label, its a model, accessibilityText may fit better.

Reply:
I don't necessarily agree 🤓😄 "Label" has two meanings, one is how it is used in `UILabel` -> A thing that has something written on it. The other meaning is "label" as the name or description for something. That's how it's used for the `.accessibilityLabel` of the `UIAccessibility` protocol. So I think it's totally valid to have `accessibilityLabel` as a property name in a model. But I won't block if you still prefer `accessibiliyText` 😄

"But that won't match with the whole model." Not sure what you mean by that.

Reply:
I initially suggested accessibilityText too (feels more like a model attribute) because that is primarily a text which one can assign to a label. Again, it’s a matter of taste.

Reply:
I don't think a model should reflect what might be the use of it. I'll change it to `accessibiliyText` - this sounds like the best solution to me.